react/InfosCarrousel/styles.styl

```diff
@@ -12,16 +13,20 @@
 
 .InfosCarrousel-navigation
     position absolute
-    bottom 0
-    right 0
+    bottom 10px
+    right 8px
     display flex
     align-items center
 
+    +mobile()
+        bottom 4px
+        right 0
+
 .InfosCarrousel-arrow
     background none
     border 0
     min-width 32px
```

Comment:
This can be replaced by `spacing_values.xl`

Reply:
Are we sure that a min-width should be implemented with a spacing value ?